docs/data-sources/cloud_l4_l7_native_load_balancer.md

```diff
@@ -70,4 +70,5 @@ data "aci_cloud_l4_l7_native_load_balancer" "example" {
 * `trunking` - (Read-Only) For virtual devices, if a trunking port group is to be used. Type: String.
 * `cloud_l4l7_load_balancer_type` - (Read-Only) Type of the Cloud L4-L7 Native Load Balancer object. Type: String.
 * `relation_cloud_rs_ldev_to_cloud_subnet` - (Read-Only) Represents the relation to a Relation from Cloud LDev to Cloud Subnet (class cloudSubnet). Type: List.
-* `aaa_domain_dn` - (Read-Only) Represents the relation to a Relation from AAA Domain to Cloud L4L7 Native Load Balancer (class aaaRbacAnnotation). Type: List.
\ No newline at end of file
+* `aaa_domain_dn` - (Read-Only) Represents the relation to a Relation from AAA Domain to Cloud L4L7 Native Load Balancer (class aaaRbacAnnotation). Type: List.
+* `static_ip_address` - (Read-Only) Static IP Address of the Cloud L4-L7 Native Load Balancer object. Type: Set of String.
```

Comment:
Should it be something like below? Set sort of already implicates uniqueness so not sure that is needed.

Furthermore I do not see the usage of Type: Set of String, but Type: List in other resources. Should we keep this consistent? Comment applies to other docs in this PR also.

```suggestion
* `static_ip_address` - (Read-Only) A list of unique Static IP Address of the Cloud L4-L7 Native Load Balancer object. Type: List.
```

Reply:
resolved